### PR TITLE
Fix/trial expired modal

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

File: src/scss/core/utilities.scss [hunk 1]
```diff
@@ -83,6 +83,10 @@ All utilities have a prefix u_ added to them
   border-radius: $border-radius-base;
 }
 
+.u_no-border {
```
Comment (RV-Adrian) [inline src/scss/core/utilities.scss]: @fjvallarino 
I added a few utility classes on your branch here. These will be applied on stage-1 as well.

Comment (fjvallarino) [reply src/scss/core/utilities.scss]: @RV-Adrian awesome! Thanks!

**Comment by fjvallarino** [comment]
@Rise-Vision/apps please review

**Comment by RV-Adrian** [comment]
@fjvallarino 👍

**Comment by fjvallarino** [comment]
@Rise-Vision/apps please review. The new changes are in: https://github.com/Rise-Vision/common-header/pull/788/commits/cedb5ebbe163d15f341995ff69dee72f3369c0c2

File: src/js/components/plans/ctr-plans-modal.js [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
       $loading.start("plans-modal");
 
-      return planFactory.getCompanyPlanStatus()
-        .then(function (allPlansMap) {
-          _allPlansMap = allPlansMap;
-          return planFactory.getPlansDetails();
-        })
-        .then(function (plans) {
-          $scope.plans = plans;
+      return $q.all([planFactory.getCompanyPlanStatus(), planFactory.getPlansDetails()])
```
Comment (alex-deaconu) [inline src/js/components/plans/ctr-plans-modal.js]: @fjvallarino nice refactoring here 👍

File: src/js/components/plans/ctr-plans-modal.js [hunk 3]
```diff
@@ -46,11 +48,34 @@ angular.module("risevision.common.components.plans")
...
+
+    $scope.isFree = function (plan) {
+      return plan.type === "free";
+    };
+
+    $scope.currentButtonVisible = function (plan) {
+      var freeOnCurrenExpired = $scope.isFree(plan) && planFactory.isTrialExpired();
+      var currentSubscribed = $scope.isCurrentPlan(plan) && $scope.isSubscribed(plan);
+
+      return freeOnCurrenExpired || currentSubscribed;
```
Comment (alex-deaconu) [inline src/js/components/plans/ctr-plans-modal.js]: @fjvallarino typo in this variable name here. Thanks.

**Pull request merged**